packages/visual-editor/src/types/types.ts

```diff
@@ -101,3 +101,6 @@ export type RichText = {
   html?: string;
   json?: string;
 };
+
+// GalleryImageType: for gallery images, only url is allowed
```

Comment:
Does this comment basically mean it is constant value only (can't come from an entity image) so it's just a url?